src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -44,11 +40,9 @@ case class WorkflowDescriptor(id: WorkflowId,
                               backend: Backend,
                               configCallCaching: Boolean,
                               lookupDockerHash: Boolean,
-                              gcsInterface: Try[GoogleCloudStorage],
-                              ioManager: IoInterface,
+                              implicit val fileSystems: List[FileSystem],
```

Comment:
`PathString.UriString` requires an implict `List[FileSystem]` that must show up automagically.